cmake/Modules/FindTBB.cmake

```diff
@@ -41,10 +41,15 @@ if(NOT TBB_LIBRARY)
          # create an imported target that also links the thread libraries via its interface
          set(TBB_BUILT_STATIC_LIB 1)
          find_package( Threads )
+         list(APPEND linkinterface "${CMAKE_THREAD_LIBS_INIT}")
+         find_library(LIBRT rt)
+         if(LIBRT)
+            list(APPEND linkinterface "${LIBRT}")
+         endif()
          add_library(tbb STATIC IMPORTED)
          set_target_properties(tbb PROPERTIES
             IMPORTED_LOCATION "${TBB_LIBRARY}"
-            IMPORTED_LINK_INTERFACE_LIBRARIES "${CMAKE_THREAD_LIBS_INIT}")
+            INTERFACE_LINK_LIBRARIES ${linkinterface})
```

Comment:
why did you change IMPORTED_LINK_INTERFACE_LIBRARIES for INTERFACE_LINK_LIBRARIES here?

Reply:
it says in the documentation that the former is deprecated and that one should be used instead

Reply:
should not make any difference as their semantics are the same, except that INTERFACE_LINK_LIBRARIES is not only for imported targets but also static library targets.

Reply:
The mechanism with using this list variable should be tested though because I remember we added the quotes due to some build issues. I think this should be correct regarding quotes and how things are added.